orthw

```diff
@@ -479,6 +479,7 @@ report() {
     --refresh-resolutions \
     --repository-configuration-file $repository_configuration_file \
     --resolutions-file $ort_config_resolutions_file \
+    -O CycloneDX=output.file.formats=json,xml \
```

Comment:
What's the reason for choosing JSON (over YML) ?

Reply:
question was YML not XML, however. Let's just go ahead. It's analog to the gitlab ci defaults now...

Reply:
CycloneDX reporter uses https://github.com/CycloneDX/cyclonedx-core-java which only supports JSON and XML